### ISSUE TITLE
SUGGESTION sell/loss tolerance: allow manual limit order thru PT (or any alternative equivalent to that action)

### ISSUE THREAD

**Issue by woodyalien** [taniman/profit-trailer#697]
Here goes a suggestion for development:

Scenario:
Today I wanted to sell one particular coin stuck in DCA with -15% for 24h. 
Using technical analysis I could see a worse forecast, it would not recover any soon - I needed to accept the loss, get rid of it and move on... **but there was room to try closing the position by -10%.**

Options:
Seems like what I wanted to do is not supported (yet):
- if I panicsell the pair, it would sell at -15%; 
- if I set stoploss it would also sell at current price; 
- if I sell thru the exchange, PT will lose track of it and damage its metrics because not showing into sales log (simply vanishes from PT);
- I also imagined setting this pair's profit to -10, but  states that profit must be positive (yeah, makes sense!), so also not an option  (but it could be an easy way!)

Request:
In other words, please consider developing something to allow manually setting on PT a particular sell order, or a sell tolerance despite the loss.

===========================

Extra:
My suggestion addresses a scenario that needs solution. Hopefully we will never need it, but it must be available when needed, as today. It regards to an instant/immediate action.
Here are other suggestions that might reduce the risk of reaching my scenario: https://github.com/taniman/profit-trailer/issues/613
https://github.com/taniman/profit-trailer/issues/691
But once reaching my scenario, an option like mine is needed to get out of it.

**Comment by woodyalien** [comment]
Well, I explained why creating an order on the exchange is not desirable, and this is part of the issue I am reporting here once being known doesn't address it...
but you suggested "use pending with the bot" - what you mean by that?  Ins't pending only what I submit thru the exchange? So what can be done manually that still goes to the sales log?

**Comment by woodyalien** [comment]
Would you please consider allowing set a negative-profit for specific pairs? This would be easy implementation and would address it!

**Comment by kenttater** [comment]
@YoJeff 

I second the motion for this suggested enhancement.  In down market times, there should be an ability for the bot to knowingly trade a pair at a loss.  I have tried to do it by setting profit, trailing, etc.. settings specific to the pair, but it does not work.  I am assuming because the math logic in PT is expecting positive values for those settings and unable to properly calculate an intentional loss.  PT should have the ability to manually set a targeted negative profit for a specific pair as well as perform proper trailing and min profit on it.  Please unclose this issue.
